globus_sdk/__init__.py

```diff
@@ -1,39 +1,46 @@
 import logging
 
 
+from globus_sdk.version import __version__
+
 from globus_sdk.response import GlobusResponse, GlobusHTTPResponse
-from globus_sdk.transfer import TransferClient
-from globus_sdk.transfer.data import TransferData, DeleteData
-from globus_sdk.auth import (
-    AuthClient, NativeAppAuthClient, ConfidentialAppAuthClient)
+
+from globus_sdk.exc import (
+    GlobusError, GlobusAPIError, TransferAPIError, SearchAPIError,
+    NetworkError, GlobusConnectionError, GlobusTimeoutError)
 
 from globus_sdk.authorizers import (
     NullAuthorizer, BasicAuthorizer, AccessTokenAuthorizer,
     RefreshTokenAuthorizer, ClientCredentialsAuthorizer)
 
-from globus_sdk.exc import GlobusError, GlobusAPIError, TransferAPIError
-from globus_sdk.exc import (
-    NetworkError, GlobusConnectionError, GlobusTimeoutError)
+from globus_sdk.auth import (
+    AuthClient, NativeAppAuthClient, ConfidentialAppAuthClient)
 
-from globus_sdk.version import __version__
+from globus_sdk.transfer import TransferClient
+from globus_sdk.transfer.data import TransferData, DeleteData
+
+from globus_sdk.search import SearchClient, SearchQuery
+
+
+__all__ = (
+    "__version__",
 
-__all__ = ["GlobusResponse", "GlobusHTTPResponse",
+    "GlobusResponse", "GlobusHTTPResponse",
 
-           "TransferClient", "TransferData", "DeleteData",
+    "GlobusError", "GlobusAPIError", "TransferAPIError", "SearchAPIError",
+    "NetworkError", "GlobusConnectionError", "GlobusTimeoutError",
+    "GlobusOptionalDependencyError",
```

Comment:
`GlobusOptionalDependencyError` isn't actually availed from `globus_sdk.exc` above.

```
>>> from globus_sdk import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'globus_sdk' has no attribute 'GlobusOptionalDependencyError'
```

(This looks like this was broken before you rearranged this though.)